tests/db/test_bigchain_api.py

```diff
@@ -257,9 +257,38 @@ def test_read_transaction_invalid_block(self, b, user_vk, user_sk):
         response = b.get_transaction(tx.id)
 
         # should be None, because invalid blocks are ignored
+        # and a copy of the tx is not in the backlog
         assert response is None
 
     @pytest.mark.usefixtures('inputs')
+    def test_read_transaction_invalid_block_and_backlog(self, b, user_vk, user_sk):
+        from bigchaindb.models import Transaction
+
+        input_tx = b.get_owned_ids(user_vk).pop()
+        input_tx = b.get_transaction(input_tx.txid)
+        inputs = input_tx.to_inputs()
+        tx = Transaction.transfer(inputs, [user_vk], input_tx.asset)
+        tx = tx.sign([user_sk])
```

Comment:
Just a note, not a blocker to merge this PR.

I wonder whether it is really needed to write a "real" tx for this test, as it seems all we would need is a "dummy" tx, just to make sure the block in-validation can take place.